### PR TITLE
Update to view templates

### PR DESCRIPTION
**Pull request by Sammyskills**
1. Corrected the wrong route name used in the magic link email template which resulted in a blank url when sent.
2. Updated the magic link form response to display an error if a wrong or invalid email address is entered.

### CODE REVIEW

File: src/Views/magic_link_form.php [hunk 1]
```diff
@@ -8,6 +8,10 @@
     <div class="card col-5 shadow-sm">
         <div class="card-body">
             <h5 class="card-title mb-5"><?= lang('Auth.useMagicLink') ?></h5>
+            
+            <?php if (session('error') !== null) : ?>
+			    <div class="alert alert-danger" role="alert"><?= session('error') ?></div>
+			<?php endif ?>
```
Comment (MGatner) [inline src/Views/magic_link_form.php:14]:
```suggestion
            
            <?php if (session('error') !== null) : ?>
            <div class="alert alert-danger" role="alert"><?= session('error') ?></div>
            <?php endif ?>
```

**Review by MGatner — CHANGES_REQUESTED** [review]
Looks good! You see to have mixed tabs and spaces in the view, quick change.

**Review by MGatner — APPROVED** [review]

File: src/Views/email_activate_email.php [hunk 2]
```diff
@@ -1,3 +1,3 @@
-<p>Please click the link below to activate your account and start using the site.</p>
+<p>Please use the code below to activate your account and start using the site.</p>
 
-<p><a href="<?= site_url('/auth/a/verify?token=' . $code) ?>"><?= site_url('/auth/a/verify') ?></a></p>
+<p>$code</p>
```
Comment (MGatner) [inline src/Views/email_activate_email.php:3]: This should be echoed out. 
```suggestion
<p><?= $code ?></p>
```
But also: why the drastic change? Was the other solution not working?

**Review by MGatner — CHANGES_REQUESTED** [review]

**Comment by Sammyskills** [comment]
Thanks for the correction.

Yes, the url from the email returns a 404 error. Moreover the `email_activate_show` form indicates the message below:

`We just sent an email to you with a code to confirm your email address. Copy that code and paste it below`

**Review by MGatner — APPROVED** [review]
Thanks for clarifying!

**Comment by kenjis** [comment]
The test fails.
```
There was 1 failure:

1) Tests\Controllers\ActionsTest::testEmailActivateShow
Failed asserting that '<!-- DEBUG-VIEW START 1 /Users/kenji/work/codeigniter/codeigniter-shield/src/Views/email_activate_email.php -->\n
<p>Please use the code below to activate your account and start using the site.</p>\n
\n
<p>367129</p>\n
\n
<!-- DEBUG-VIEW ENDED 1 /Users/kenji/work/codeigniter/codeigniter-shield/src/Views/email_activate_email.php -->' contains "Please click the link below to activate your account".

.../codeigniter-shield/tests/Controllers/ActionsTest.php:220
```

**Comment by MGatner** [comment]
You're right - tests/Controllers/ActionsTest.php L221 needs updating.

**Comment by kenjis** [comment]
I sent a PR: #93

**Pull request merged**